src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -5039,6 +5108,7 @@ internal static bool IsSynthesizedAttributeOptional(WellKnownMember attributeMem
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_IteratorStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncIteratorStateMachineAttribute__ctor:
+                case WellKnownMember.Microsoft_VisualBasic_Conversion__FixDecimal:
```

Comment:
>case WellKnownMember.Microsoft_VisualBasic_Conversion__FixDecimal: [](http://example.com/codeflow?start=16&length=66)

This change doesn't look right. This member is not an attribute. And even if it was, the optimization has nothing to do with synthesizing attributes, I think.

Reply:
I will revert this change later.